### PR TITLE
Fire up BadgeWorkers from DependencyWorker. Resolves #11

### LINKED ISSUE #11: When to fire up badge workers?
Currently letting the app sleep for 5s before running the BadgeWorkers for the first time, in order to give the DependencyWorkers a chance to catch up. That applies to the "first run" of the Workers. On the scheduler run, the BadgeWorkers are just planned 5 minutes later than the Dependency Workers. @bartkamphorst in #4:

> Either the badge workers should wait for data to be stored (either by polling the store every so often, or, preferably, through some kind of callback), or the badge job should be scheduled/fired up by the dependency workers.

Sounds like a good idea! But, in the latter case: if a DependencyWorker throws an unhandled exception, that would mean the Badge Worker also doesn't get called (and so the SVG would remain the same as before the DepWorker threw the exception). Should we handle this by just catching all exceptions in DepWorker#perform and setting dependencies to `nil` in that case?

On the first run, just waiting for the store to fill up is a possibility, although in that case, if the DepWorkers run into a problem, the app wouldn't start forever.

### PR DESCRIPTION
**Pull request by dometto**
Proof of concept: spin up the badge workers from the DependencyWorker, as suggested in https://github.com/repotag/cutting_edge/issues/11

Seems to be working fine!

### CODE REVIEW

File: lib/cutting_edge/workers/dependency.rb [hunk 1]
```diff
@@ -10,18 +10,23 @@ class DependencyWorker < GenericWorker
 
   # Order is significant for purposes of calculating results[:outdated]
   STATUS_TYPES = [:outdated_major, :outdated_minor, :outdated_patch, :ok, :no_requirement, :unknown]
-  OUTDATED_TYPES = STATUS_TYPES[0..-3]
+  OUTDATED_TYPES = STATUS_TYPES[0..-4]
```
Comment (dometto) [inline lib/cutting_edge/workers/dependency.rb]: Caught a bug; `:ok` status was being treated as outdated. More specs needed :)

File: lib/cutting_edge/app.rb [hunk 2]
```diff
@@ -35,6 +38,7 @@ def worker_fetch(repo)
 
 module CuttingEdge
    LAST_VERSION_TIMEOUT = 5
+   BADGE_WORK_QUEUE =
```
Comment (bartkamphorst) [inline lib/cutting_edge/app.rb]: This is incomplete.

Comment (dometto) [reply lib/cutting_edge/app.rb]: Ah, thanks -- that's a remnant of me trying to implement a Queue, as per your other suggestion, and then realizing I had no idea how to. :)

File: spec/fixtures.rb [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+      :ok=>3,
+      :outdated=>:outdated_major,
+      :outdated_major=>3,
+      :outdated_minor=>1,
+      :outdated_patch=>0,
+      :outdated_total=>4,
+      :unknown=>0,
+      :no_requirement => 0
+    },
+    'gollum-updated' =>
```
Comment (dometto) [inline spec/fixtures.rb:60]: Same as `'gollum'`, but with all the latest versions increased 1 full level.

File: bin/cutting_edge [hunk 4]
```diff
@@ -4,11 +4,14 @@ require File.expand_path('../../lib/cutting_edge/app.rb', __FILE__)
 require 'yaml'
 require 'rufus-scheduler'
 
+REFRESH_SCHEDULE = '1h'
```
Comment (dometto) [inline bin/cutting_edge:7]: Will move all defaults to a separate file in another PR

**Pull request merged**